examples/test/model/index.html

```diff
@@ -11,13 +11,16 @@
       <a-assets>
```

Comment:
Is that on latest? We pushed some text patches switching to raw shader material.